components/brave_wallet_ui/components/buy-send-swap/shared-styles.ts

```diff
@@ -30,11 +31,8 @@ export const SelectScrollSearchContainer = styled.div`
   justify-content: flex-start;
   overflow-y: scroll;
   overflow-x: hidden;
-  position: absolute;
-  top: 96px;
-  bottom: 18px;
-  left: 18px;
-  right: 18px;
+  width: 100%;
+  max-height: 100%;
```

Comment:
Since this is a shared style, this change breaks other parts of the wallet.

![Screen Shot 2022-09-28 at 3 45 27 PM](https://user-images.githubusercontent.com/40611140/192886439-cf72b93e-f7cd-4cf7-9479-54e27362b1e1.png)